R/zchunk_batch_ssp15_emissions_factors_xml.R

```diff
@@ -24,6 +24,10 @@ module_emissions_batch_ssp15_emissions_factors_xml <- function(command, ...) {
 
     # ===================================================
 
+    # Rename L251.ssp15_ef column to match the L2 data names
+    # in order to meet the requirements for the xml header names.
+    L251.ssp15_ef <- rename(L251.ssp15_ef, `emiss.coef` = `emiss.coeff`)
```

Comment:
Why does this fix / how does this related to the timeshift text?

Reply:
Agh it was somewhat of a guess but in short is was a necessary change.

The driver failed to run on my local because the header used in the `add_xml_data` requires an `emiss.coef` instead of an  `emiss.coeff`. I thought that the error might have been misdiagnosed on travis as a time shift error. 

I checked in with @pralitp and he said that I could rename the `emiss.coef` column in the L2 Data Names list but I was afraid of the number of files that has correctly used `emiss.coef`. Otherwise I `NULL` as the third argument in the function assuming that the order of the columns is correct. I don't want to make assumptions when it comes to the xmls.

Reply:
Well, mad props to you. Thanks.